contracts/ERC-721/IERC721Pool.sol

```diff
@@ -45,6 +60,25 @@ interface IERC721Pool {
     /// @param ids The asset token IDs sent from the user's account to the pool.
     function deposit(uint256[] calldata ids) external;
 
+    /// @notice Allows the factory to rescue the last NFT in the pool and set the pool to frozen.
+    ///
```

Comment:
@surbhiaudichya The emitted event is not mentioned here

Reply:
@scorpion9979 
As I understand it, by initializing the `uint256 public nonce` variable with an initial value of zero and incrementing it for each subsequent pool creation, we can still compute the pool address off-chain based on the value of the nonce. However, I would like to understand your perspective more clearly to ensure we are on the same page.

Reply:
Thanks @scorpion9979, I will update NatSpec comment.